packaging/conda/meta.yaml

```diff
@@ -36,16 +36,57 @@ requirements:
   run:
     - {{ mpi }}
 
+# requirements:
+#   run:
+#     - khiops-core
+#     - kni
+#     - kni-transfer
+
 outputs:
   - name: khiops-core
+    version: {{ os.environ.get('KHIOPS_VERSION') }}
+    requirements:
+      build:
+        - cmake
+        - ninja
+        - {{ compiler('cxx') }}
+      host:
+        - {{ mpi }}
+      run:
+        - {{ mpi }}
+    files:
+      - bin/MODL*
     test:
       commands:
         - MODL -v
         - MODL_Coclustering -v
+  - name: kni
+    version: {{ os.environ.get('KHIOPS_VERSION') }}
+    files: 
+      - bin/libKhiopsNativeInterface* # [linux or osx]
```

Comment:
Why the glob here? Is the extension on MacOS different from Linux?

Reply:
yes I think it is dylib

Reply:
Update: on macOS the versioning follows libname1.2.3.dylib [doc here](https://docstore.mik.ua/orelly/unix3/mac/ch05_04.htm)

Reply:
OK, and OK with your solution with the `*.dylib` glob.